### PR TITLE
PIC-2560: Alert users when they will lose a comment or note

### PR DESCRIPTION
**Pull request by rschallapalli**

### CODE REVIEW

File: server/views/court-case-progress.njk [hunk 1]
```diff
@@ -1,6 +1,23 @@
 <div class="govuk-grid-row" id="caseProgressComponent">
     <div class="govuk-grid-column-two-thirds">
         <h2 class="govuk-heading-m" id="caseHearingsHeading">Case progress</h2>
+
+        <div id="popup-wrapper" class="popup-toggle">
```
Comment (rschallapalli) [inline server/views/court-case-progress.njk:5]: The size and position are way off
Cypress tests to be updated for the popup

File: integration-tests/integration/features/case-progress.feature [hunk 2]
```diff
@@ -95,3 +96,39 @@ Feature: Case progress
...
+      | 12th hearing        | Saturday 14 December 2019, Court 2, morning session, Leicester    |                 |
+      | 8th hearing         | Wednesday 14 August 2019, Court 2, morning session, Leicester     |                 |
+      | 7th hearing         | Sunday 14 July 2019, Court 1, morning session, Leicester          |                 |
+      | 5th hearing         | Tuesday 14 May 2019, Court 2, morning session, North Shields      |                 |
+
+
+    And the note with the id "1288880" on hearing "2aa6f5e0-f842-4939-bc6a-01346abc09e3" is filled with the text "I am a first unsaved note"
+    And the note with the id "123650" on hearing "2aa6f5e0-f842-4939-bc6a-01346abc09e3" is filled with the text "A"
+
+    Then should appears a css hidden modal which displays information
```
Comment (rschallapalli) [inline integration-tests/integration/features/case-progress.feature:130]: avoid technical words like css where possible? "Then user should be alerted with a popup"? 
If you are keeping it as is please correct appears -> appear

File: server/views/components/hearing-notes/template.njk [hunk 3]
```diff
@@ -21,21 +21,19 @@
         </table>
     </div>
 
-    <form id="save-notes-{{ params.hearing.hearingId }}" action="summary/notes" method="post">
+    <form id="save-notes-{{ params.hearing.hearingId }}" class="comments-form-tag" action="summary/notes" method="post">
```
Comment (rschallapalli) [inline server/views/components/hearing-notes/template.njk:24]: Can remove class="comments-form-tag" as we are not selecting forms any more?

File: server/views/court-case-comments.njk [hunk 4]
```diff
@@ -25,8 +25,8 @@
                     These comments will not be saved to NDelius.
                 </p>
             </div>
-            <form method="post" action="summary/comments">
-                <textarea class="govuk-textarea"
+            <form method="post" id="case-comment-form" class="comments-form-tag" action="summary/comments">
```
Comment (rschallapalli) [inline server/views/court-case-comments.njk:28]: same here, can remove class="comments-form-tag"

File: integration-tests/integration/features/case-progress.feature [hunk 5]
```diff
@@ -9,9 +9,12 @@ Feature: Case progress
     And I click the "Accept analytics cookies" button
     Then I should not see the cookie banner
 
+    Given I am an authenticated user
```
[the remark this answers is not in the document]
Comment (sarahyjja) [reply integration-tests/integration/features/case-progress.feature:12]: You are right

**Pull request merged**